### ISSUE TITLE
ChargePointMaxProfile not respected if charging with multiple connectors

### ISSUE THREAD

**Issue by rpseng** [matth-x/MicroOcppSimulator#21]
Hello, thanks again for this amazing tool.

As per the OCPP 1.6 spec (section 3.13.1, pg 20)  `ChargePointMaxProfile` can only be set at Charge Point ConnectorId 0. Additionally, it should limit the power or current to be shared by all connectors of the Charge Point. What I have experienced is that  `ChargePointMaxProfile` is currently being applied in a per connector basis and not shared by all connectors.

If I set a limit of 5000 W, the total load is 10000 W:
![image](https://github.com/matth-x/MicroOcppSimulator/assets/5142420/6b79b1cf-58b5-44e4-ba14-d124b493df13)

Perhaps more important is to check if the same occurs with MicroOcpp.

**Comment by matth-x** [comment]
True, good finding! I missed that part completely when creating the Simulator. It needs an additional internal load balancer to handle ChargePointMaxProfiles.

This indeed reflects the behavior of the MicroOcpp core library. MicroOcpp does not come with an internal load balancer, but assumes that the host firmware implements it (which would be the Simulator here). I definitely need to point that out in the documentation there too, thanks for the reminder.
